_delphi_utils_python/delphi_utils/geomap.py

```diff
@@ -102,8 +106,10 @@ def __init__(self):
         """
         self.crosswalk_filepaths = CROSSWALK_FILEPATHS
         self.crosswalks = {
-            "zip": {"fips": None, "hrr": None, "msa": None, "pop": None, "state": None},
-            "fips": {"zip": None, "hrr": None, "msa": None, "pop": None, "state": None},
+            "zip": {"fips": None, "hrr": None, "msa": None,
```

Comment:
Are these always going to have `None` as the value?  Might be more scalable/futureproof to write as a dictionary comprehension:
```
"zip": {x:None for x in ["fips",...]}
```

Reply:
I think so, unless we end up preloading crosswalks sometime? I'll add your suggestion.